tests/model_monitoring/test_tdengine.py

```diff
@@ -237,3 +238,127 @@ def test_get_records_query(
             )
             == expected_query
         )
+
+    @pytest.mark.parametrize(
+        (
+            "subtable",
+            "columns_to_filter",
+            "start",
+            "end",
+            "timestamp_column",
+            "interval",
+            "limit",
+            "agg",
+            "sliding_window_step",
+        ),
+        [
+            (
+                "subtable_1",
+                ["column2"],
+                datetime.datetime.now().astimezone() - datetime.timedelta(hours=2),
+                datetime.datetime.now().astimezone() - datetime.timedelta(hours=1),
```

Comment:
In tests it's OK, but note that the difference between the start and the end time here is a bit longer than 1hr.
It's always better to use `end = mlrun.utils.datetime_now()` and then subtract a `timedelta` to get the start time.